### PR TITLE
Fix typo and punctuation

### PR DESCRIPTION
**Pull request by Joe-Moran**
## DESCRIBE YOUR PR
Fixed a typo and added punctuation to description for consistency.

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [X] None: Not urgent, can wait up to 1 week+

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: docs/platforms/dotnet/common/configuration/http-client-errors.mdx [hunk 1]
```diff
@@ -30,7 +30,7 @@ let httpClient = new HttpClient(new SentryHttpMessageHandler())
 
 <Alert title="Tip">
 
-You can skip the above step when using any of the Sentry .NET SDKs that are registered via dependency injection (`Senry.AspNetCore`, `Sentry.Maui`, `Sentry.Extensions.Logging`, etc.).
+You can skip the above step when using any of the Sentry .NET SDKs that are registered via dependency injection (`Sentry.AspNetCore`, `Sentry.Maui`, `Sentry.Extensions.Logging`, etc.).
```
Comment (lizokm) [inline docs/platforms/dotnet/common/configuration/http-client-errors.mdx:33]:
```suggestion
You can skip the above step when using any of the Sentry .NET SDKs that are registered via dependency injection (`Sentry.AspNetCore`, `Sentry.Maui`, `Sentry.Extensions.Logging`, and so on).
```

**Comment by lizokm** [comment]
Thanks for catching and for contributing to our docs @Joe-Moran!

**Review by lizokm — APPROVED** [review]
Thanks for the update!

**Comment by Joe-Moran** [comment]
> Thanks for catching and for contributing to our docs @Joe-Moran!

@lizokm Woot! Happy to help in this very small way.

**Pull request merged**